framework/contrib/hit/parse.cc

```diff
@@ -17,7 +19,16 @@
 namespace hit
 {
 
-const std::string indentString = "  ";
+// returns the type of quoting used on string s (i.e. " or ') or an empty string otherwise.
+std::string
```

Comment:
Consider a return type of `char`.

Reply:
I did, but the contexts where I call this function often need a std::string too - so I just decided on string.  This is an internal function that isn't an exposed API.

Reply:
Thanks!